.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @bimtauer
```

Comment:
I added this in my fork as a precaution against hijacking since I did test-publish the python [package rdb-py](https://pypi.org/project/rdb-py/) from there. I used a trusted publisher configured for my fork for this. Publishing the python bindings wont work from here. Let me know if you also want to publish 0.3 of the python lib from here first. 
Sorry for not mentioning this ealier.

Reply:
All good! Once I publish the latest version on crates.io, I will hand it over to you and your repo becomes the main one. So we can keep it as that and you can do the rdb-py release afterwards.